app/src/main/java/com/yarendemirkaya/shoppinglist/adapter/ShoppingItemAdapter.kt

```diff
@@ -31,6 +37,19 @@ class ShoppingItemAdapter(
 
     override fun onBindViewHolder(holder: ShoppingViewHolder, position: Int) {
         val curShoppingItem = items[position]
+        val builder: AlertDialog.Builder = AlertDialog.Builder(holder.itemView.context) //?
+        builder
+            .setMessage("Do you really want to delete this item")
+            .setTitle("Delete Item")
+            .setPositiveButton("YES") { dialog, which ->
+                    viewModel.delete(curShoppingItem)
+
+            }
+            .setNegativeButton("NO") { dialog, which ->
+                dialog.dismiss()
```

Comment:
currentItem in amount unu = 1 yapmalisin yoksa aslinda arka tarafta 0 oluyor ama ui a islemiyor.